lib/ui/locations/infoj.mjs

```diff
@@ -28,7 +28,7 @@ export default (location, infoj_order) => {
   groups = {}
 
   // The infoj_order may be assigned to the layer.
-  infoj_order ??= location.layer.infoj_order
+  infoj_order ??= location?.layer?.infoj_order
```

Comment:
Shouldn't a location always have a layer??

Reply:
Yes, it should. But better safe than sorry.

Reply:
Alrighty!

Reply:
I wasn't 100% sure on this - I thought this was the case and should always have a layer and thought better to be safe!

Reply:
Gazetteer locations may not have a layer I think.